demos/appyx-navigation/ios/build.gradle.kts

```
@@ -0,0 +1,61 @@
+plugins {
+    id("com.bumble.appyx.multiplatform")
+    kotlin("multiplatform")
+    kotlin("native.cocoapods")
+    id("org.jetbrains.compose")
+    id("com.google.devtools.ksp")
+}
+
+kotlin {
+
+    iosX64()
+    iosArm64()
+    iosSimulatorArm64()
+
+    cocoapods {
+        version = "1.0.0"
+        summary = "appyx-nvaigation ios module"
+        homepage = "https://bumble-tech.github.io/appyx/interactions/"
```

Wrong url (I assume copy-paste leftover)

```suggestion
        homepage = "https://bumble-tech.github.io/appyx/navigation/"
```